src/topology/order/lower_topology.lean

```diff
@@ -0,0 +1,230 @@
+/-
+Copyright (c) 2023 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import topology.order.lattice
+
+/-!
+# Lower topology
+
+This file introduces the lower topology on a preorder as the topology generated by the complements
+of the closed intervals to infinity.
+
+## Main statements
+
+- `t0_space` - the lower topology on a partial order is T₀
+- `is_topological_basis` - the complements of the upper closures of finite subsets form a basis for
+  the lower topology
```

Comment:
You need to prepend namespaces here, otherwise they will not be hyperlinked to the appropriate declarations in mathlib docs.

Reply:
Ah yes, done, thanks.